### ISSUE TITLE
sfsl should not check buckets from installed packages, but just currently added buckets

### ISSUE THREAD

**Issue by brian6932** [winpax/sfsu#8]
If a package is kept while the original bucket is removed, then `sfsl` throws an error
```pwsh
❯ sfsl
Error: failed to resolve path '~\scoop\buckets\knox-scoop': The system cannot find the file specified.
; class=Os (2); code=NotFound (-3)
```
Instead it should just check the dirs in `"$($env:SCOOP_GLOBAL ?? $env:SCOOP ?? '~\scoop')\buckets"`

**Comment by jewlexx** [comment]
I believe it is trying to open that dir in order to find the remote of the bucket that said package is from. I'll have a look when I have the chance

**Comment by jewlexx** [comment]
Fixed as of v1.1.1
